.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Is this important here to use ? I have seen this for some Migration based chores in GitHub afaik

Reply:
Yeah @at-the-vr as the server fails to start. I don't know why.
Here's the error :
```bash
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - SWC minify release candidate enabled. https://nextjs.link/swcmin
warn  - Attempted to load @next/swc-linux-x64-gnu, but an error occurred: libssl.so.1.1: cannot open shared object file: No such file or directory
warn  - Attempted to load @next/swc-linux-x64-gnux32, but it was not installed
warn  - Attempted to load @next/swc-linux-x64-musl, but it was not installed
warn  - Error: EROFS: read-only filesystem, mkdir '/node_modules/next/wasm'
    at makeError$1 (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22607:24)
    at EROFS (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22634:10)
    at ZipFS.mkdirSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24546:13)
    at ZipFS.mkdirpSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:22828:16)
    at ZipFS.mkdirSync (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24544:19)
    at ZipFS.mkdirPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:24540:17)
    at /home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25607:26
    at /home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25789:81
    at ZipOpenFS.getZipPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25896:22)
    at ZipOpenFS.makeCallPromise (/home/saptarshi-sarkar/AccessibleWebDev/.pnp.cjs:25789:23) {
  code: 'EROFS'
}
error - Failed to load SWC binary for linux/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
```

Reply:
@EmmaDawsonDev @at-the-vr Please let me know if the file is actually related to the error or not.